codex/erasure/erasure.nim

```diff
@@ -338,16 +343,16 @@ proc encodeData(
 proc encode*(
   self: Erasure,
   manifest: Manifest,
-  blocks: int,
-  parity: int): Future[?!Manifest] {.async.} =
+  blocks: uint,
+  parity: uint): Future[?!Manifest] {.async.} =
```

Comment:
Blocks is converted to an int here. Perhaps it'd make sense to use a `Natural` or even make a new `NoBlocks` int type?

Reply:
Blocks are really the K param for the erasure coding, so uint should be fine.

Reply:
Actually, `Natural` is probably better.